sale_blanket_order/models/__init__.py

```diff
@@ -1,3 +1,5 @@
-from . import blanket_orders
-from . import sale_orders
+from . import blanket_order
+from . import blanket_order_line
+from . import sale_order
+from . import sale_order_line
```

Comment:
Also, commit [[IMP] sale_blanket_order](https://github.com/OCA/sale-workflow/pull/2509/commits/b74168dafc58103447878ed6360c49f415643510) should be part of the migration commit no?

Reply:
> Are we forced to split these files?

we should split these files because each python file should contain one class.

> Also, commit [[IMP] sale_blanket_order](https://github.com/OCA/sale-workflow/pull/2509/commits/b74168dafc58103447878ed6360c49f415643510) should be part of the migration commit no?

The migration commit was added by **Jasper Jumelet**, i have to add this commit separately to keep track

Reply:
in fact, i don't split them but @jasperjumelet.
The commit [[IMP] sale_blanket_order](https://github.com/OCA/sale-workflow/pull/2509/commits/b74168dafc58103447878ed6360c49f415643510) contains all the changes i did.

Reply:
Yes I know it's not in your commit, but as soon as someone is opening a PR for a migration, he is taking the responsibility of it, and as such this PR can't be merged IMO :)
For a renaming, you can take the right to edit the commit of @jasperjumelet as soon as you keep the other changes he did.

Or start from the beginning with your own commit(s) if you think it's easier.

Reply:
ok, wip

Reply:
@sebalix i have updated the migration commit, please review